src/md4c.c

```diff
@@ -5659,7 +5664,7 @@ md_line_indentation(MD_CTX* ctx, unsigned total_indent, OFF beg, OFF* p_end)
     return indent - total_indent;
 }
 
-static const MD_LINE_ANALYSIS md_dummy_blank_line = { MD_LINE_BLANK, 0 };
+static const MD_LINE_ANALYSIS md_dummy_blank_line = { MD_LINE_BLANK, 0, 0, 0, 0 };
```

Comment:
If you prefer implicit zero initialization I _think_ you could specify `-std=c11` (or `-std=c99` even.) AFAIK it was undefined behavior prior to C99. I might be wrong. Anyhow, simply initializing these seemed like the smallest and simplest fix.

Reply:
FWIW, standard C has always had implicit zero initialization for braced initializers; even in C89. GCC complains because apparently it's considered "error prone" to not warn for missing elements, except for the special case of `{0}`.